install/kubernetes/helm/istio/values.yaml

```diff
@@ -85,7 +85,10 @@ global:
   disablePolicyChecks: false
 
   # EnableTracing sets the value with same name in istio config map, requires pilot restart to take effect.
-  enableTracing: true
+  tracing:
+    enabled: true
+    namespace: istio-system
+    service: jaeger
```

Comment:
Please give out a comment and example for external jaeger service here.

By external, do you mean jaeger outside of istio-system ns or jaeger outside of the k8s cluster?

Reply:
I believe he's referring to a jaeger installation elsewhere in the cluster.

He should have probably just used a single variable for the complete address, though, so that either one would be supported.

(also hoping this PR gets re-opened.)

Reply:
A similar change has already been merged - see https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L207-L210

@visheyra Thanks for the PR, but I think it can be closed.